pkg/gcp/resources.go

```diff
@@ -53,35 +57,74 @@ type resourceNetworkInfo struct {
 	SubnetUrl   string
 	ResourceID  string
 	NetworkName string
+	Address     string
+}
+
+type ServiceAttachmentDescription struct {
```

Comment:
Since the definition of `ServiceAttachmentDescription` is defined in paraglider scope and not GCP, may be we need to define it somewhere, when we document private endpoint support. Just noting it here :)

Reply:
Yeah, we definitely should include it in the docs about this.